Formula/s/squiid.rb

```diff
@@ -0,0 +1,48 @@
+class Squiid < Formula
+  desc "Modular calculator written in Rust"
+  homepage "https://imaginaryinfinity.net/"
```

Comment:
It seems that the project homepage is at https://imaginaryinfinity.net/projects/squiid/ precisely.